packages/pn-personagiuridica-webapp/src/components/IntegrazioneApi/VirtualKeys.tsx

```diff
@@ -152,7 +152,7 @@ const VirtualKeys: React.FC = () => {
 
       {!issuerState.tosAccepted ? (
```

Comment:
Add an OR condition here. The information about the presence of the issuer is contained into the issuerState.issuer object

Reply:
done